Soccar [UI]/src/nl/soccar/ui/fx/drawable/CarUiFx.java

```diff
@@ -102,4 +109,37 @@ private void drawWheel(WheelPhysics wheel, GraphicsContext gc) {
         gc.restore();
     }
 
+    private void drawBoostTrail(List<Point2D> trail, GraphicsContext gc) {
+
+        double prevX, prevY, x, y;
+        double o = 0.0;
+        int r = 30;
+        int g = 30;
+        int b = 230;
+
+        // Cancel when there is no trail.
+        if (trail.size() == 0) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1155)